### ISSUE TITLE
[com.riotgames.league.wildrift] Unable to proceed with the game on v0.3.5.240913

### ISSUE THREAD

**Issue by flame-0** [microg/GmsCore#2678]
**Application**
Name: Wild Rift
Package id: com.riotgames.league.wildrift

**Description**
In v0.3.5.240913, I can't proceed with the game whether AssetModuleService is enabled or disabled.
```
$ grep -i wildrift log.cat 
12-11 13:45:30.426  1880  2916 I ActivityTaskManager: START u0 {act=android.intent.action.MAIN cat=[android.intent.category.LAUNCHER] flg=0x10000000 pkg=com.riotgames.league.wildrift cmp=com.riotgames.league.wildrift/com.tencent.lolm.lgame} with LAUNCH_SINGLE_TASK from uid 10135 (BAL_ALLOW_VISIBLE_WINDOW) result code=0
12-11 13:45:30.427  2939  2982 V WindowManagerShell: Transition requested (#179): android.os.BinderProxy@2d969fa TransitionRequestInfo { type = OPEN, triggerTask = TaskInfo{userId=0 taskId=41 displayId=0 isRunning=true baseIntent=Intent { act=android.intent.action.MAIN cat=[android.intent.category.LAUNCHER] flg=0x10000000 pkg=com.riotgames.league.wildrift cmp=com.riotgames.league.wildrift/com.tencent.lolm.lgame } baseActivity=ComponentInfo{com.riotgames.league.wildrift/com.tencent.lolm.lgame} topActivity=ComponentInfo{com.riotgames.league.wildrift/com.tencent.lolm.lgame} origActivity=null realActivity=ComponentInfo{com.riotgames.league.wildrift/com.tencent.lolm.lgame} numActivities=1 lastActiveTime=1570419 supportsMultiWindow=true resizeMode=2 isResizeable=true minWidth=-1 minHeight=-1 defaultMinSize=220 token=WCT{android.window.IWindowContainerToken$Stub$Proxy@6387bab} topActivityType=1 pictureInPictureParams=null shouldDockBigOverlays=false launchIntoPipHostTaskId=-1 lastParentTaskIdBeforePip=-1 displayCutoutSafeInsets=Rect(0, 95 - 0, 0) topActivityInfo=ActivityInfo{cfcf208 com.tencent.lolm.lgame} launchCookies=[] positionInParent=Point(0, 0) parentTaskId=-1 isFocused=false isVisible=false isVisibleRequested=false isSleeping=false locusId=null displayAreaFeatureId=1 isTopActivityTransparent=false appCompatTaskInfo=AppCompatTaskInfo { topActivityInSizeCompat=false topActivityEligibleForLetterboxEducation= false isLetterboxDoubleTapEnabled= false topActivityEligibleForUserAspectRatioButton= false topActivityBoundsLetterboxed= false isFromLetterboxDoubleTap= false topActivityLetterboxVerticalPosition= -1 topActivityLetterboxHorizontalPosition= -1 topActivityLetterboxWidth=1080 topActivityLetterboxHeight=2400 isUserFullscreenOverrideEnabled=false isSystemFullscreenOverrideEnabled=false cameraCompatControlState=hidden}}, pipTask = null, remoteTransition = null, displayChange = null, flags = 0, debugId = 179 }
12-11 13:45:30.437  1880  2008 I ActivityManager: Start proc 13054:com.riotgames.league.wildrift/u0a161 for next-top-activity {com.riotgames.league.wildrift/com.tencent.lolm.lgame}
12-11 13:45:30.443 13054 13054 I league.wildrift: Using CollectorTypeCC GC.
12-11 13:45:30.446 13054 13054 E league.wildrift: Not starting debugger since process cannot load the jdwp agent.
12-11 13:45:30.447  1880 10511 D CoreBackPreview: Window{3708ee9 u0 Splash Screen com.riotgames.league.wildrift}: Setting back callback OnBackInvokedCallbackInfo{mCallback=android.window.IOnBackInvokedCallback$Stub$Proxy@5ce0459, mPriority=0, mIsAnimationCallback=false}
12-11 13:45:30.461  1880  1990 V WindowManager: Sent Transition (#179) createdAt=12-11 13:45:30.420 via request=TransitionRequestInfo { type = OPEN, triggerTask = TaskInfo{userId=0 taskId=41 displayId=0 isRunning=true baseIntent=Intent { act=android.intent.action.MAIN cat=[android.intent.category.LAUNCHER] flg=0x10000000 pkg=com.riotgames.league.wildrift cmp=com.riotgames.league.wildrift/com.tencent.lolm.lgame } baseActivity=ComponentInfo{com.riotgames.league.wildrift/com.tencent.lolm.lgame} topActivity=ComponentInfo{com.riotgames.league.wildrift/com.tencent.lolm.lgame} origActivity=null realActivity=ComponentInfo{com.riotgames.league.wildrift/com.tencent.lolm.lgame} numActivities=1 lastActiveTime=1570419 supportsMultiWindow=true resizeMode=2 isResizeable=true minWidth=-1 minHeight=-1 defaultMinSize=220 token=WCT{RemoteToken{5518d15 Task{ce5cc4b #41 type=standard A=10161:com.riotgames.league.wildrift}}} topActivityType=1 pictureInPictureParams=null shouldDockBigOverlays=false launchIntoPipHostTaskId=-1 lastParentTaskIdBeforePip=-1 displayCutoutSafeInsets=Rect(0, 95 - 0, 0) topActivityInfo=ActivityInfo{1dd191a com.tencent.lolm.lgame} launchCookies=[] positionInParent=Point(0, 0) parentTaskId=-1 isFocused=false isVisible=false isVisibleRequested=false isSleeping=false locusId=null displayAreaFeatureId=1 isTopActivityTransparent=false appCompatTaskInfo=AppCompatTaskInfo { topActivityInSizeCompat=false topActivityEligibleForLetterboxEducation= false isLetterboxDoubleTapEnabled= false topActivityEligibleForUserAspectRatioButton= false topActivityBoundsLetterboxed= false isFromLetterboxDoubleTap= false topActivityLetterboxVerticalPosition= -1 topActivityLetterboxHorizontalPosition= -1 topActivityLetterboxWidth=1080 topActivityLetterboxHeight=2400 isUserFullscreenOverrideEnabled=false isSystemFullscreenOverrideEnabled=false cameraCompatControlState=hidden}}, pipTask = null, remoteTransition = null, displayChange = null, flags = 0, debugId = 179 }
12-11 13:45:30.461  1880  1990 V WindowManager:         {WCT{RemoteToken{5518d15 Task{ce5cc4b #41 type=standard A=10161:com.riotgames.league.wildrift}}} m=OPEN f=NONE leash=Surface(name=Task=41)/@0x64b68ff sb=Rect(0, 0 - 1080, 2400) eb=Rect(0, 0 - 1080, 2400) d=0 endFixedRotation=1},
12-11 13:45:30.471 13054 13054 D nativeloader: Configuring clns-4 for other apk /system/framework/org.apache.http.legacy.jar. target_sdk_version=33, uses_libraries=ALL, library_path=/data/app/~~_y75Fu8FXj5_A9PwUG9F4Q==/com.riotgames.league.wildrift-kCkM6Fcb42JWpGrjo6Bnbg==/lib/arm64:/data/app/~~_y75Fu8FXj5_A9PwUG9F4Q==/com.riotgames.league.wildrift-kCkM6Fcb42JWpGrjo6Bnbg==/base.apk!/lib/arm64-v8a:/data/app/~~_y75Fu8FXj5_A9PwUG9F4Q==/com.riotgames.league.wildrift-kCkM6Fcb42JWpGrjo6Bnbg==/split_config.arm64_v8a.apk!/lib/arm64-v8a:/data/app/~~_y75Fu8FXj5_A9PwUG9F4Q==/com.riotgames.league.wildrift-kCkM6Fcb42JWpGrjo6Bnbg==/split_pad1.apk!/lib/arm64-v8a, permitted_path=/data:/mnt/expand:/data/user/0/com.riotgames.league.wildrift
12-11 13:45:30.475 13054 13054 W ziparchive: Unable to open '/data/app/~~_y75Fu8FXj5_A9PwUG9F4Q==/com.riotgames.league.wildrift-kCkM6Fcb42JWpGrjo6Bnbg==/split_config.arm64_v8a.dm': No such file or directory
12-11 13:45:30.475 13054 13054 W ziparchive: Unable to open '/data/app/~~_y75Fu8FXj5_A9PwUG9F4Q==/com.riotgames.league.wildrift-kCkM6Fcb42JWpGrjo6Bnbg==/split_config.arm64_v8a.dm': No such file or directory
12-11 13:45:30.476 13054 13054 W league.wildrift: Entry not found
12-11 13:45:30.476 13054 13054 W ziparchive: Unable to open '/data/app/~~_y75Fu8FXj5_A9PwUG9F4Q==/com.riotgames.league.wildrift-kCkM6Fcb42JWpGrjo6Bnbg==/split_pad1.dm': No such file or directory
12-11 13:45:30.476 13054 13054 W ziparchive: Unable to open '/data/app/~~_y75Fu8FXj5_A9PwUG9F4Q==/com.riotgames.league.wildrift-kCkM6Fcb42JWpGrjo6Bnbg==/split_pad1.dm': No such file or directory
12-11 13:45:30.477 13054 13054 W league.wildrift: Entry not found
12-11 13:45:30.477 13054 13054 D nativeloader: Configuring clns-5 for other apk /data/app/~~_y75Fu8FXj5_A9PwUG9F4Q==/com.riotgames.league.wildrift-kCkM6Fcb42JWpGrjo6Bnbg==/base.apk:/data/app/~~_y75Fu8FXj5_A9PwUG9F4Q==/com.riotgames.league.wildrift-kCkM6Fcb42JWpGrjo6Bnbg==/split_config.arm64_v8a.apk:/data/app/~~_y75Fu8FXj5_A9PwUG9F4Q==/com.riotgames.league.wildrift-kCkM6Fcb42JWpGrjo6Bnbg==/split_pad1.apk. target_sdk_version=33, uses_libraries=, library_path=/data/app/~~_y75Fu8FXj5_A9PwUG9F4Q==/com.riotgames.league.wildrift-kCkM6Fcb42JWpGrjo6Bnbg==/lib/arm64:/data/app/~~_y75Fu8FXj5_A9PwUG9F4Q==/com.riotgames.league.wildrift-kCkM6Fcb42JWpGrjo6Bnbg==/base.apk!/lib/arm64-v8a:/data/app/~~_y75Fu8FXj5_A9PwUG9F4Q==/com.riotgames.league.wildrift-kCkM6Fcb42JWpGrjo6Bnbg==/split_config.arm64_v8a.apk!/lib/arm64-v8a:/data/app/~~_y75Fu8FXj5_A9PwUG9F4Q==/com.riotgames.league.wildrift-kCkM6Fcb42JWpGrjo6Bnbg==/split_pad1.apk!/lib/arm64-v8a, permitted_path=/data:/mnt/expand:/data/user/0/com.riotgames.league.wildrift
12-11 13:45:30.518 13054 13863 W league.wildrift: Unsupported class loader
12-11 13:45:30.522 13054 13054 D nativeloader: Load /data/app/~~_y75Fu8FXj5_A9PwUG9F4Q==/com.riotgames.league.wildrift-kCkM6Fcb42JWpGrjo6Bnbg==/lib/arm64/liblz4.so using ns clns-5 from class loader (caller=<unknown>): ok
12-11 13:45:30.522 13054 13054 D nativeloader: Load /data/app/~~_y75Fu8FXj5_A9PwUG9F4Q==/com.riotgames.league.wildrift-kCkM6Fcb42JWpGrjo6Bnbg==/lib/arm64/liblzma.so using ns clns-5 from class loader (caller=<unknown>): ok
12-11 13:45:30.523 13054 13054 D nativeloader: Load /data/app/~~_y75Fu8FXj5_A9PwUG9F4Q==/com.riotgames.league.wildrift-kCkM6Fcb42JWpGrjo6Bnbg==/lib/arm64/libzipw.so using ns clns-5 from class loader (caller=<unknown>): ok
12-11 13:45:30.524 13054 13054 D nativeloader: Load /data/app/~~_y75Fu8FXj5_A9PwUG9F4Q==/com.riotgames.league.wildrift-kCkM6Fcb42JWpGrjo6Bnbg==/lib/arm64/libTDataMaster.so using ns clns-5 from class loader (caller=<unknown>): ok
12-11 13:45:30.526 13054 13054 D nativeloader: Load /data/app/~~_y75Fu8FXj5_A9PwUG9F4Q==/com.riotgames.league.wildrift-kCkM6Fcb42JWpGrjo6Bnbg==/lib/arm64/libMSDKCore.so using ns clns-5 from class loader (caller=<unknown>): ok
12-11 13:45:30.635 13054 13893 D nativeloader: Load /data/app/~~_y75Fu8FXj5_A9PwUG9F4Q==/com.riotgames.league.wildrift-kCkM6Fcb42JWpGrjo6Bnbg==/lib/arm64/libOodler.so using ns clns-5 from class loader (caller=<unknown>): ok
12-11 13:45:30.636 13054 13893 I lgame   : isSoFilesReady file not ok filePath = /data/user/0/com.riotgames.league.wildrift/files/app_libs/libtdr.so
12-11 13:45:30.646 13054 13863 W league.wildrift: Unsupported class loader
12-11 13:45:30.646 13054 13863 W league.wildrift: Unsupported class loader
12-11 13:45:30.656  1880  2916 D CoreBackPreview: Window{8e68071 u0 com.riotgames.league.wildrift/com.tencent.lolm.lgame}: Setting back callback OnBackInvokedCallbackInfo{mCallback=android.window.IOnBackInvokedCallback$Stub$Proxy@f7cf3c4, mPriority=0, mIsAnimationCallback=false}
12-11 13:45:30.740  1880  1990 I ActivityTaskManager: Displayed com.riotgames.league.wildrift/com.tencent.lolm.lgame for user 0: +320ms
12-11 13:45:30.820  1880  2916 I ImeTracker: com.riotgames.league.wildrift:111b96b1: onRequestHide at ORIGIN_SERVER reason HIDE_UNSPECIFIED_WINDOW fromUser false
12-11 13:45:30.821  1880  2916 I ImeTracker: com.riotgames.league.wildrift:111b96b1: onCancelled at PHASE_SERVER_SHOULD_HIDE
12-11 13:45:30.825  3681  3681 I GoogleInputMethodService: GoogleInputMethodService.onStartInput():1911 onStartInput(EditorInfo{EditorInfo{packageName=com.riotgames.league.wildrift, inputType=0, inputTypeString=NULL, enableLearning=false, autoCorrection=false, autoComplete=false, imeOptions=0, privateImeOptions=null, actionName=UNSPECIFIED, actionLabel=null, initialSelStart=-1, initialSelEnd=-1, initialCapsMode=0, label=null, fieldId=0, fieldName=null, extras=null, hintText=null, hintLocales=[]}}, false)
12-11 13:45:30.826  1880  4616 W PackageConfigPersister: App-specific configuration not found for packageName: com.riotgames.league.wildrift and userId: 0
12-11 13:45:31.043  1880  1990 V WindowManager:         {WCT{RemoteToken{5518d15 Task{ce5cc4b #41 type=standard A=10161:com.riotgames.league.wildrift}}} m=CHANGE f=NONE p=WCT{RemoteToken{85bdea9 DefaultTaskDisplayArea@65358178}} leash=Surface(name=Task=41)/@0x64b68ff sb=Rect(0, 0 - 1080, 2400) eb=Rect(0, 0 - 2400, 1080) d=0 r=0->1:0},
12-11 13:45:31.046  1880  8835 D CoreBackPreview: Window{3708ee9 u0 Splash Screen com.riotgames.league.wildrift EXITING}: Setting back callback null
12-11 13:45:31.049  1880  4616 W InputManager-JNI: Input channel object '3708ee9 Splash Screen com.riotgames.league.wildrift (client)' was disposed without first being removed with the input manager!
12-11 13:45:31.058  3681  3681 I GoogleInputMethodService: GoogleInputMethodService.onStartInput():1911 onStartInput(EditorInfo{EditorInfo{packageName=com.riotgames.league.wildrift, inputType=0, inputTypeString=NULL, enableLearning=false, autoCorrection=false, autoComplete=false, imeOptions=0, privateImeOptions=null, actionName=UNSPECIFIED, actionLabel=null, initialSelStart=-1, initialSelEnd=-1, initialCapsMode=0, label=null, fieldId=0, fieldName=null, extras=null, hintText=null, hintLocales=[]}}, true)
12-11 13:45:31.059  1880  3173 W PackageConfigPersister: App-specific configuration not found for packageName: com.riotgames.league.wildrift and userId: 0
12-11 13:45:31.153 13054 13896 W InteractionJankMonitor: Initializing without READ_DEVICE_CONFIG permission. enabled=false, interval=1, missedFrameThreshold=3, frameTimeThreshold=64, package=com.riotgames.league.wildrift
12-11 13:45:31.310  5571  5571 D GmsGcmRegister: handleMessage: package=com.riotgames.league.wildrift what=1 id=1
12-11 13:45:31.339  5571 13923 D GmsHttpFormClient: device=4004207708204478511&app=com.riotgames.league.wildrift&cert=13baa399838d4710d9c6a94525382100c6e71984&app_ver=829664&X-subtype=972597317697&X-app_ver=829664&X-osv=34&X-cliv=fcm-23.0.8&X-gmsv=240913010&X-appid=foWNKi_pRy-qKqWbzDz3pT&X-scope=*&X-Goog-Firebase-Installations-Auth=eyJhbGciOiJFUzI1NiIsInR5cCI6IkpXVCJ9.eyJhcHBJZCI6IjE6OTcyNTk3MzE3Njk3OmFuZHJvaWQ6OTIzM2ZhMjdmMWRlNGUzY2VkMzZjMCIsImV4cCI6MTczNDUwMDcyOSwiZmlkIjoiZm9XTktpX3BSeS1xS3FXYnpEejNwVCIsInByb2plY3ROdW1iZXIiOjk3MjU5NzMxNzY5N30.AB2LPV8wRgIhANtR6cV7LxPhDby9krXQnD7Ccu7-2Ih-xib4lIQ9tjq8AiEA7sJWUQv7oB_SvuNQfRXHGx9s-_AEy3wbHJDsdimnqsM&X-gmp_app_id=1%3A972597317697%3Aandroid%3A9233fa27f1de4e3ced36c0&X-firebase-app-name-hash=R1dAH9Ui7M-ynoznwBdw01tLxhI&X-app_ver_name=5.3.0.8296&target_ver=33&sender=972597317697
12-11 13:45:34.082 13054 13937 I CopySoFilesRunnable: CopySoFilesRunnable run soFile: /data/app/~~_y75Fu8FXj5_A9PwUG9F4Q==/com.riotgames.league.wildrift-kCkM6Fcb42JWpGrjo6Bnbg==/lib/arm64/libtdr_cp.so destFile: /data/user/0/com.riotgames.league.wildrift/files/app_libs/libtdr_cp.so
12-11 13:45:34.082 13054 13937 I Oodle   : jni decompressFile called! input file is /data/user/0/com.riotgames.league.wildrift/files/app_libs/libtdr_cp.so
12-11 13:45:34.083 13054 13937 I Oodle   : jni decompressFile called! output file is /data/user/0/com.riotgames.league.wildrift/files/app_libs/libtdr.so
12-11 13:45:34.090 13054 13937 I CopySoFilesRunnable: CopySoFilesRunnable decompressResult ok: /data/user/0/com.riotgames.league.wildrift/files/app_libs/libtdr.so
12-11 13:45:34.092 13054 13937 I CopySoFilesRunnable: CopySoFilesRunnable run soFile: /data/app/~~_y75Fu8FXj5_A9PwUG9F4Q==/com.riotgames.league.wildrift-kCkM6Fcb42JWpGrjo6Bnbg==/lib/arm64/libLNet_cp.so destFile: /data/user/0/com.riotgames.league.wildrift/files/app_libs/libLNet_cp.so
12-11 13:45:34.092 13054 13937 I Oodle   : jni decompressFile called! input file is /data/user/0/com.riotgames.league.wildrift/files/app_libs/libLNet_cp.so
12-11 13:45:34.092 13054 13937 I Oodle   : jni decompressFile called! output file is /data/user/0/com.riotgames.league.wildrift/files/app_libs/libLNet.so
12-11 13:45:34.100 13054 13937 I CopySoFilesRunnable: CopySoFilesRunnable decompressResult ok: /data/user/0/com.riotgames.league.wildrift/files/app_libs/libLNet.so
12-11 13:45:34.109 13054 13937 I CopySoFilesRunnable: CopySoFilesRunnable run soFile: /data/app/~~_y75Fu8FXj5_A9PwUG9F4Q==/com.riotgames.league.wildrift-kCkM6Fcb42JWpGrjo6Bnbg==/lib/arm64/libFEProj_cp.so destFile: /data/user/0/com.riotgames.league.wildrift/files/app_libs/libFEProj_cp.so
12-11 13:45:34.109 13054 13937 I Oodle   : jni decompressFile called! input file is /data/user/0/com.riotgames.league.wildrift/files/app_libs/libFEProj_cp.so
12-11 13:45:34.109 13054 13937 I Oodle   : jni decompressFile called! output file is /data/user/0/com.riotgames.league.wildrift/files/app_libs/libFEProj.so
12-11 13:45:34.279 13054 13937 I CopySoFilesRunnable: CopySoFilesRunnable decompressResult ok: /data/user/0/com.riotgames.league.wildrift/files/app_libs/libFEProj.so
12-11 13:45:34.301 13054 13937 I CopySoFilesRunnable: CopySoFilesRunnable run soFile: /data/app/~~_y75Fu8FXj5_A9PwUG9F4Q==/com.riotgames.league.wildrift-kCkM6Fcb42JWpGrjo6Bnbg==/lib/arm64/libil2cpp_cp.so destFile: /data/user/0/com.riotgames.league.wildrift/files/app_libs/libil2cpp_cp.so
12-11 13:45:34.302 13054 13937 I Oodle   : jni decompressFile called! input file is /data/user/0/com.riotgames.league.wildrift/files/app_libs/libil2cpp_cp.so
12-11 13:45:34.302 13054 13937 I Oodle   : jni decompressFile called! output file is /data/user/0/com.riotgames.league.wildrift/files/app_libs/libil2cpp.so
12-11 13:45:34.789 13054 13937 I CopySoFilesRunnable: CopySoFilesRunnable decompressResult ok: /data/user/0/com.riotgames.league.wildrift/files/app_libs/libil2cpp.so
12-11 13:45:34.795 13054 13937 I CopySoFilesRunnable: CopySoFilesRunnable run soFile: /data/app/~~_y75Fu8FXj5_A9PwUG9F4Q==/com.riotgames.league.wildrift-kCkM6Fcb42JWpGrjo6Bnbg==/lib/arm64/libunity_cp.so destFile: /data/user/0/com.riotgames.league.wildrift/files/app_libs/libunity_cp.so
12-11 13:45:34.795 13054 13937 I Oodle   : jni decompressFile called! input file is /data/user/0/com.riotgames.league.wildrift/files/app_libs/libunity_cp.so
12-11 13:45:34.795 13054 13937 I Oodle   : jni decompressFile called! output file is /data/user/0/com.riotgames.league.wildrift/files/app_libs/libunity.so
12-11 13:45:34.832 13054 13937 I CopySoFilesRunnable: CopySoFilesRunnable decompressResult ok: /data/user/0/com.riotgames.league.wildrift/files/app_libs/libunity.so
12-11 13:45:34.833 13054 13054 I lgame   : onFinishSoFiles customNativeDirPath = /data/user/0/com.riotgames.league.wildrift/files/app_libs retCode = 1
12-11 13:45:34.833 13054 13054 I lgame   : onFinishSoFiles customNativeDirPath = /data/user/0/com.riotgames.league.wildrift/files/app_libs
12-11 13:45:34.835 13054 13054 D nativeloader: Load /data/app/~~_y75Fu8FXj5_A9PwUG9F4Q==/com.riotgames.league.wildrift-kCkM6Fcb42JWpGrjo6Bnbg==/lib/arm64/libmain.so using ns clns-5 from class loader (caller=<unknown>): ok
12-11 13:45:34.835 13054 13054 I UnityPlayer: loadNative CustomNativeDirPath > 0 is: /data/user/0/com.riotgames.league.wildrift/files/app_libs
12-11 13:45:34.833 13054 13054 W league.wildrift: type=1400 audit(0.0:1701): avc:  granted  { execute } for  path="/data/data/com.riotgames.league.wildrift/files/app_libs/libunity.so" dev="dm-47" ino=109824 scontext=u:r:untrusted_app_32:s0:c161,c256,c512,c768 tcontext=u:object_r:app_data_file:s0:c161,c256,c512,c768 tclass=file app=com.riotgames.league.wildrift
12-11 13:45:34.842 13054 13054 I UnityPlayer: initJni CustomNativeDirPath > 0 is: /data/user/0/com.riotgames.league.wildrift/files/app_libs
12-11 13:45:35.892 13054 13938 I Unity   : ApplicationInfo com.riotgames.league.wildrift version 5.3.0.8296 build b91d4749-d1fb-47ad-8f88-069a3c2916e3
12-11 13:45:35.889 13054 13054 W UnityMain: type=1400 audit(0.0:1702): avc:  granted  { execute } for  path="/data/data/com.riotgames.league.wildrift/files/app_libs/libil2cpp.so" dev="dm-47" ino=34530 scontext=u:r:untrusted_app_32:s0:c161,c256,c512,c768 tcontext=u:object_r:app_data_file:s0:c161,c256,c512,c768 tclass=file app=com.riotgames.league.wildrift
12-11 13:45:36.193 13054 13938 V Native  : Open and map /storage/emulated/0/Android/data/com.riotgames.league.wildrift/files/il2cpp/Metadata/global-metadata.dat success!
12-11 13:45:36.529 13054 13938 D league.wildrift: PlayerBase::PlayerBase()
12-11 13:45:36.529 13054 13938 D league.wildrift: TrackPlayerBase::TrackPlayerBase()
12-11 13:45:36.532  1282 11655 I UsecaseValidator: verifyAudioAttributes update usage: 1 to AUDIO_USAGE_GAME for output: 13 pid: 13054 package: com.riotgames.league.wildrift
12-11 13:45:37.514 13054 13942 W Unity   : AndroidVideoMedia::OpenExtractor could not translate jar:file:///data/app/~~_y75Fu8FXj5_A9PwUG9F4Q==/com.riotgames.league.wildrift-kCkM6Fcb42JWpGrjo6Bnbg==/base.apk!/assets/assetpack/Video/Compliance/loginLoop.mp4 to local file. Make sure file exists, is on disk (not in memory) and not compressed.
12-11 13:45:37.866  3681  3681 I GoogleInputMethodService: GoogleInputMethodService.onStartInput():1911 onStartInput(EditorInfo{EditorInfo{packageName=com.riotgames.league.wildrift, inputType=0, inputTypeString=NULL, enableLearning=false, autoCorrection=false, autoComplete=false, imeOptions=0, privateImeOptions=null, actionName=UNSPECIFIED, actionLabel=null, initialSelStart=-1, initialSelEnd=-1, initialCapsMode=0, label=null, fieldId=0, fieldName=null, extras=null, hintText=null, hintLocales=[]}}, false)
12-11 13:45:37.866  1880 10511 W PackageConfigPersister: App-specific configuration not found for packageName: com.riotgames.league.wildrift and userId: 0
12-11 13:45:38.497  3301  3301 D CarrierSvcBindHelper: onPackageModified: com.riotgames.league.wildrift
12-11 13:45:38.518  3301  3515 D ImsResolver: maybeAddedImsService, packageName: com.riotgames.league.wildrift
```

**Reproduce**
1. Install Wild Rift via Aurora Store
2. Open Wild Rift
3. An error popup appears: "Unknown error. -5"

https://github.com/user-attachments/assets/62185219-0ab4-4f26-a26e-1fa835dbee4b

**Expected behavior**
The game should launch without errors and proceed to download and decompress the necessary files.

**System**
Android Version: Android 14
Custom ROM: LineageOS 21

**microG**
microG Core version: v0.3.5.240913
microG Self-Check results: All ticked

**Additional context**
Running LineageOS 21 with privileged microG integrated at build time.

**Comment by ale5000-git** [comment]
AssetModuleService was added in 0.3.5 so in 0.3.4 you don't have it.
You have to update both microG Services and microG Companion.

**Comment by flame-0** [comment]
> AssetModuleService was added in 0.3.5 so in 0.3.4 you don't have it. You have to update both microG Services and microG Companion.

What do you mean I have to update both? I always update them in my [repository](https://codeberg.org/vyrine/android_vendor_microg).
![Image](https://github.com/user-attachments/assets/dc27685d-1c8a-4f49-9e39-7466b0d65581)
![Image](https://github.com/user-attachments/assets/a35c3fe6-46ae-460b-b975-49cb3aa4a424)

**Comment by ale5000-git** [comment]
I only said that because you wrote v0.3.4 in the title and in the description.

**Comment by Nuyoah-Yc** [comment]
The PR of https://github.com/microg/GmsCore/pull/2621 needs to be merged, and the game needs to be synchronized with Android

**Comment by flame-0** [comment]
Fixed in [v0.3.6.244735](https://github.com/microg/GmsCore/releases/tag/v0.3.6.244735)
